test-infra/airflow/deployment.yaml

```diff
@@ -0,0 +1,51 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: airflow
+spec:
+  selector:
+    name: airflow
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 8080
+
+---
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: airflow
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: airflow
+    spec:
+      containers:
+      - name: airflow
+        image: gcr.io/mlkube-testing/airflow:latest
+        # What arguments do we need to set to use the LocalExecutor
+        command:
+          # Tells entrypoint.sh to start the webserver
+          - /entrypoint.sh
+          - webserver
+        env:
+        - name: POSTGRES_HOST
+          value: localhost
+        ports:
+        - containerPort: 8080
+        # TODO(jlewi): Should we store the postgres data on a PD to make it
+        # invariant to disruptions.
+      - name: postgres
+        image: postgres:9.6
```

Comment:
10.0 is out now :-)

Reply:
also suggest to use `alpine` tag which is much smaller

Reply:
Done.